.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: node_js
 node_js:
   - 6
-  - 7
```

Comment:
upgrading to eslint 5.16.0 required to fix security vulnerability does not support node 7

Reply:
Let's mark this as breaking and just move to `6.X` for eslint. We'll hold off on merging until after knowing usage data.